doc/how_to/custom_components/reactive_esm/reactive_esm_layout.md

```diff
@@ -0,0 +1,143 @@
+# Create Layouts With ReactiveESM
+
+In this guide we will show you how to build custom layouts using HTML and `ReactiveESM`.
+
+## Layout a single Panel Component
+
+You can layout a single Panel component as follows.
+
+:::::{tab-set}
+
+::::{tab-item} JavaScript
+
+```{pyodide}
+import param
```

Comment:
How do I `.watch` for changes to the children such that I can update if the `object` changes?

Reply:
Oh. It seem the `render` function is rerun when a `child` on `children` is changed. That holds true for js, `html` and `react` implementations.